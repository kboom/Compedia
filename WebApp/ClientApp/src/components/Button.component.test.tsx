import renderer from "react-test-renderer";
import { Button } from "./Button.component";

it("renders correctly when there are no tasks", () => {
  const tree = renderer.create(<Button>test string</Button>).toJSON();
  expect(tree).toMatchSnapshot();
});
