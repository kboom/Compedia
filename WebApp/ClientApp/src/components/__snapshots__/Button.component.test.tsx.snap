// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no tasks 1`] = `
<button
  className="button"
  disabled={false}
>
  <strong>
    test string
  </strong>
</button>
`;
